app.rb

```diff
@@ -128,7 +128,7 @@ def json_only
       end
 
       search_params = Search::QueryParameters.new(parser.parsed_params)
-      searcher = Search::Query.new(unified_index, registries)
+      searcher = Search::Query.new(unified_index, registries, metasearch_index: Rummager.search_config.metasearch_index)
```

Comment:
feel like we could nuke quite a bit more of this file so that we have a more consistent approach to search from web and rake tasks.